src/Microsoft.DotNet.ApiSymbolExtensions/AssemblySymbolLoaderFactory.cs

```diff
@@ -8,7 +8,14 @@ namespace Microsoft.DotNet.ApiSymbolExtensions
     /// </summary>
     public sealed class AssemblySymbolLoaderFactory : IAssemblySymbolLoaderFactory
     {
+        public AssemblySymbolLoaderFactory(bool includeInternals = false)
+        {
+            IncludeInternals = includeInternals;
+        }
+
+        public bool IncludeInternals { get; }
```

Comment:
Please add this to the interface with triple slash documentation and then use inheritdoc here.

Reply:
I'll fix the doc issues for the API I'm touching but this codebase is not consistent with docs.  Many public APIs are missing docs and these projects aren't even set up to produce doc files.  If you wish to keep this project documented then docs XML should be enabled so that any missing or out-of-sync docs will show up as compiler warnings that devs can resolve independently.  That should save both of us significant time in PR review.

Reply:
Thank you. Yes, we should turn on documentation file generation and make sure that CS1591 isn't suppressed.

Reply:
There is already an issue.  I was just moving this.